Automattic/wp-calypso
# Re-enables backups on desktop app

#### Changes proposed in this Pull Request

* This re-enables Backup in the Jetpack section on the desktop app. 
* Also enables on-demand scans, which should be active.

#### Testing instructions



* See #49082 for instructions for regression testing.
* Verify UI is correct and backup download + restore work.

Fixes #49082 and 1164141197617539-as-1199604822034003.
